packages/story-editor/src/components/canvas/utils/useFullbleedMediaAsBackground.js

```diff
@@ -25,8 +33,17 @@ import { MEDIA_ELEMENT_TYPES } from '@googleforcreators/elements';
  */
 import isTargetCoveringContainer from '../../../utils/isTargetCoveringContainer';
 import { useStory, useCanvas } from '../../../app';
+import { CONFIRMATION_BACKGROUND_MESSAGE_STORAGE_KEY } from './constants';
```

Comment:
`utils` folders are usually for utils only, so the constants file would be better outside of it. Since `canvas` doesn't have its own constants file and this constant is used only in this file, I think we could just move this constant directly to this file instead of creating a separate file for it. Thoughts?

On the name -- maybe `BACKGROUND_AUTOSET_MESSAGE_STORAGE_KEY` could be a more understandable name?

Reply:
Or `DISMISS_BACKGROUND_AUTOSET_MESSAGE_STORAGE_KEY`

Reply:
Done